app/models/survey_phrase.rb

```diff
@@ -20,6 +19,6 @@ def self.mentions_by_date_range_for_phrase(phrase, start_date, end_date)
       end
     end
 
-    mentions
+    mentions.sort
```

Comment:
More out of curiosity than anything else, why this change? I expect there is a good reason but looking at the change I don't understand why it's necessary (should be functionally the same and probably faster for the db to do it). Maybe update the commit msg